pkg/jobs/trigger_event.go

```diff
@@ -44,18 +53,56 @@ func (t *EventTrigger) Valid(key, value string) bool {
 	return false
 }
 
+func (t *EventTrigger) interestedBy(e *realtime.Event) bool {
+
+	if !t.mask.Verbs.Contains(permissions.Verb(e.Type)) {
+		return false
+	}
+
+	if len(t.mask.Values) == 0 {
+		return true
+	}
+
+	if t.mask.Selector == "" {
+		return t.mask.ValuesContain(e.Doc.ID())
+	}
+
+	if v, ok := e.Doc.(permissions.Validable); ok {
+		return t.mask.ValuesValid(v)
+	}
+
+	return false
+}
+
+func addEventToMessage(e *realtime.Event, base *Message) (*Message, error) {
+	var basemsg interface{}
+	base.Unmarshal(&basemsg)
```

Comment:
Couldn't we use the `Data` field directly instead of unmarshalling/marshalling the message ? Like so:

```go
return NewMessage(JSONEncoding, map[string]interface{}{
  "message": json.RawMessage(base.Data),
  "event":   e,
})
```